views/includes/check-your-answers/individual-trustee.html

```diff
@@ -209,7 +209,7 @@
   }), rows) %}
 {% endif %}
 
-{% if not pageParams.isRegistration %}
+{% if not pageParams.isRegistration and (trustStillInvolved === "Yes" or isNotStillInvolved) %}
```

Comment:
We set this constant up on line 48 in this file:

`{% set isNotStillInvolved = stillInvolved === "No" %}`

Maybe we should set a similar one up for 'still involved'?

I'm also wondering about the check - we see if it's 'yes' or 'no' but I assume that's excluding the case of it being null/undefined?

Reply:
Updated this to be a variable. With these we are testing for a specific yes/no value.

Reply:
Correct.